frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -100,28 +110,65 @@ export const ClusterServiceVersionTableHeader = () => {
     },
     {
       title: 'Namespace',
-      sortField: 'metadata.namespace',
+      sortFunc: 'namespace',
       transforms: [sortable],
       props: { className: tableColumnClasses[1] },
     },
     {
-      title: 'Status',
+      title: 'Managed Namespaces',
+      sortFunc: 'managedNamespace',
+      transforms: [sortable, wrappable],
       props: { className: tableColumnClasses[2] },
     },
     {
-      title: 'Deployment',
-      sortField: 'spec.install.spec.deployments[0].name',
-      transforms: [sortable],
+      title: 'Status',
       props: { className: tableColumnClasses[3] },
     },
     {
-      title: 'Provided APIs',
+      title: 'Last Updated',
       props: { className: tableColumnClasses[4] },
     },
+    {
+      title: 'Provided APIs',
+      props: { className: tableColumnClasses[5] },
+    },
     {
       title: '',
+      props: { className: tableColumnClasses[6] },
+    },
+  ];
+};
+
+export const ClusterServiceVersionTableHeader = () => {
+  return [
+    {
+      title: 'Name',
+      sortField: 'metadata.name',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Managed Namespaces',
+      sortFunc: 'managedNamespace',
+      transforms: [sortable, wrappable],
+      props: { className: tableColumnClasses[2] },
```

Comment:
I think, due to copy/paste, the `props: { className: tableColumnClasses[2] },` is off by 1?  It jumps from `0` to `2` here, between `Name` and `Managed Namespace`.

Reply:
This was intentional.